.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "liveServer.settings.port": 5501
-}
\ No newline at end of file
+  "liveServer.settings.port": 5502
+}
```

Comment:
- [OPTIONAL] Since this is your own VSCode setting. it doesn't necessarily need to be committed into the repo. so having this in the .gitignore file may be a good choice 👍

Reply:
Good morning, thank you very much for your advice. @Bondok6